assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -8407,4 +8629,790 @@ tests:
+              object: document:3
+            expectation: false
+          - tuple:
+              user: user:dylan
+              relation: viewer
+              object: document:pub
+            expectation: true
+          - tuple:
+              user: user:dylan
+              relation: viewer
+              object: document:pub1
+            expectation: true
+        listObjectsAssertions:
+          - request:
+              user: user:anne
+              type: document
+              relation: viewer
+            expectation:
+              - document:1
+              - document:2
+              - document:3
+              - document:pub
+              - document:pub1
+          - request:
+              user: user:bob
+              type: document
+              relation: viewer
+            expectation:
+              - document:1
+              - document:2
+              - document:pub
+              - document:pub1
+          - request:
+              user: user:charlie
+              type: document
+              relation: viewer
+            expectation:
+              - document:1
+              - document:3
+              - document:pub
+              - document:pub1
+          - request:
+              user: user:dylan
+              type: document
+              relation: viewer
+            expectation:
+              - document:pub
+              - document:pub1
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+              - user:charlie
+          - request:
+              filters:
+                - user
+              object: document:2
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:bob
+          - request:
+              filters:
+                - user
+              object: document:3
+              relation: viewer
+            expectation:
+              - user:anne
+              - user:charlie
+          - request:
+              filters:
+                - user
+              object: document:pub
+              relation: viewer
+            expectation:
+              - user:*
+          - request:
+              filters:
+                - user
+              object: document:pub1
+              relation: viewer
+            expectation:
+              - user:*
+              - user:anne
+              - user:bob
+  - name: ttu_orphan_parent
+    stages:
+      - model: |
+          model
+            schema 1.1
+          type user
+          type group1
+            relations
+              define member: [user, user:*]
+          type group2
+            relations
+              define member: [user, user:*]
+          type document
+            relations
+              define parent: [group1, group2]
+              define viewer: member from parent
```

Comment:
this is the same model as the test preceding it, can we collapse both tests into one? in other words: the assertions on this test, are they testing something new compared to the preceding test?

Reply:
combined test into one